main.tf

```diff
@@ -94,7 +94,7 @@ resource "helm_release" "castai_agent" {
 
   set_sensitive {
     name  = "apiKey"
-    value = castai_cluster_token.cluster_token.cluster_token
```

Comment:
IMO you should also update provider version requirements https://github.com/castai/terraform-castai-eks-cluster/blob/main/versions.tf#L12

Reply:
Yes, you are right, fixed that. Now dependency on the provider defined as ">= 2.0.0"